srcpkgs/mold/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'mold'
+pkgname=mold
+version=1.0.2
+revision=1
+archs="i686 x86_64"
```

Comment:
Why limit the archs? I've built mold locally on an x86_64-musl arch. You can test build local archs with xbps-src.

Reply:
I forgot it. Sorry